### PR TITLE
Embed chatroom into team page + corrected Wasin’s controller injection mistake

### PR DESCRIPTION
**Pull request by hmhv1222**
Sorry for making it so late

### CODE REVIEW

**Comment by hmhv1222** [comment]
Embed chatroom into team page

**Comment by hmhv1222** [comment]
Previous commits included:
- Added functions of add/remove wanted skills, personalities, horoscope, team leaders, and change team leaders
- Added function of calculating number of pretty girls
- Modified save/load function

File: app/js/team.js [hunk 1]
```diff
@@ -72,125 +91,244 @@ angular.module('teamform-team-app', ['firebase'])
...
-	$scope.removeMember = function(member) {
-		
+	$scope.removeMember = function(member) 
+	{
 		var index = $scope.param.teamMembers.indexOf(member);
-		if ( index > -1 ) {
+		if ( index > -1 ) 
+		{
 			$scope.param.teamMembers.splice(index, 1); // remove that item
+			$scope.param.currentTeamSize --;
```
Comment (vivtony00) [inline app/js/team.js:215]: currentTeamSize is the maximum size of team
see line 57

File: app/js/team.js [hunk 2]
```diff
@@ -72,125 +91,244 @@ angular.module('teamform-team-app', ['firebase'])
...
+			$scope.saveFunc();
+		}
+	} 
+	
+	$scope.calculateNumPrettyGirls = function()
+	{
+		int count = 0;
+		for(int i = 0; i < $scope.param.teamMembers.length; i++)
+		{
+			if($scope.param.teamMembers[i].gender == 'F' || $scope.param.teamMembers[i].gender == 'f')
```
Comment (vivtony00) [inline app/js/team.js:316]: teamMembers will not contain information
If you need his/her gender, you should get it from profile

**Review by vivtony00 — COMMENTED** [review]
see comment inside

**Pull request merged**